internal/testing/server/build/store.go

```diff
@@ -5,42 +5,53 @@
 
 package build
 
-import "github.com/daytonaio/daytona/pkg/build"
+import t_build "github.com/daytonaio/daytona/pkg/build"
```

Comment:
I assume this alias is because you use `build` as a variable name.

I suggest removing the alias and renaming the variable.